scripts/hail_batch/project_wgs_onto_snp_chip_pca/project_wgs_samples_onto_snp_chip.py

```diff
@@ -29,19 +29,32 @@ def query():
     tob_wgs = hl.read_matrix_table(TOB_WGS)
     tob_wgs = hl.experimental.densify(tob_wgs)
     tob_wgs = tob_wgs.annotate_entries(GT=lgt_to_gt(tob_wgs.LGT, tob_wgs.LA))
-
     snp_chip = snp_chip.semi_join_rows(tob_wgs.rows())
+    snp_chip_path = output_path('snp_chip_filtered_by_tob_wgs.mt')
```

Comment:
Checkpoints should go to the temporary bucket. (Also below.)

Reply:
to add the checkpoints to the temporary bucket, would that just be `output_path('snp_chip_filtered_by_tob_wgs.mt', 'tmp')`

Reply:
Yes, that's right!